src/App.svelte

```diff
@@ -51,8 +51,68 @@
       link: "https://github.com/EddieJaoudeCommunity/hacktoberfest-practice",
     }
   ];
-</script>
 
+  const moderatorsData = [
+    {
+      name:'Stephen Mount',
+      desc: 'After 10+ years in software engineering, I am coming up with ideas and things to reverse engineer things for the new wave of ARM etc etc',
+      github:'https://github.com/stemount',
+      website:'https://ste.london/',
+      img:"https://avatars1.githubusercontent.com/u/150512",
+      alt:"@stemount"
+    },
+    {
+      name: 'Michael Mba',
+      desc:'Linux Systems Integrator | Open Source Enthusiast | Photographer | Gamer || Starting my Python journey.',
+      github:'https://github.com/mikeysan',
+      website:'https://mikeysan.hashnode.dev/',
+      img:"https://avatars0.githubusercontent.com/u/13338176?s=460&u=d66a31344752bb3fa882a59b3effb10e084f226c&v=4",
+      alt:"@mikeysan"
+        
+    },
+    {
+      name: 'Praveen Kumar ',
+      desc: 'Microsoft MVP, Web Developer and Computer Software & UX Architect. Mentor - Full Stack @Thinkful-Ed.',
+      github:'https://github.com/praveenscience',
+      website:'https://praveen.science/',
+      img:"https://avatars1.githubusercontent.com/u/1830380?s=460&u=74697d8b1cbf3e16adec7b411369afbd53ce4864&v=4",
+      alt:"@praveenscience"
+           
+    },
+    {
+      name: 'David' ,
+      desc: 'Job: Software Developer Consultant I love programming, more specifically w/ Kotlin... it\'s just the best. Very interested in Web Dev and building discord bots',
+      github:'https://github.com/BOLT04',
+      website:'https://bolt04.github.io/react-ultimate-resume/',
+      img:'https://avatars1.githubusercontent.com/u/18630253?s=460&u=36cdb692666236ff5db7aa3c71e58327bdbdecc7&v=4',
+      alt:"@BOLT04"
+    },
+    {
+      name: 'Eddie Jaoude' ,
+      desc: 'Fullstack open source DevRel! I love code and the community :). GitHub Star ⭐ program 🤓',
+      github:'https://github.com/eddiejaoude',
+      website:'http://youtube.com/eddiejaoude?sub_confirmation=1',
+      img:"https://avatars3.githubusercontent.com/u/624760?s=460&v=4",
```

Comment:
Formatting and quotes are not consistent.

Please use single quotes and have a space after the `:`

Reply:
doing it !

Reply:
done @eddiejaoude